spec/helpers/sufia_helper_spec.rb

```diff
@@ -233,16 +237,16 @@ def create_models(model, user1, user2)
         expect(subject).to include('class="glyphicon glyphicon-new-window"')
       end
     end
-    context "Hash argument for title" do # note: title typically is NOT configured with an iconify_auto_link helper
-      subject { helper.iconify_auto_link(document: document, value: text, config: blacklight_config.index_fields['title_tesim']) }
-      it { is_expected.to eq '<span>Foo &lt; <a href="http://www.example.com"><span class="glyphicon glyphicon-new-window"></span> http://www.example.com</a>. &amp; More text</span>' }
-    end
-    context "Hash argument for description" do # note: description typically IS configured with an iconify_auto_link helper
-      let(:conf) { blacklight_config.index_fields['description_tesim'] }
-      subject { helper.iconify_auto_link(document: document, value: text, config: conf) }
-      it do
-        pending 'Need a different way to test description'
-        is_expected.to eq '<span>Foo &lt; <a href="http://www.example.com"><span class="glyphicon glyphicon-new-window"></span> http://www.example.com</a>. &amp; More text</span>'
+
+    context "when using a hash argument" do
+      subject { helper.iconify_auto_link(arg) }
+      describe "auto-linking in the title" do
+        let(:arg) { { document: document, value: text, config: blacklight_config.index_fields['title_tesim'], field: 'title_tesim' } }
```

Comment:
It seems like this test is reaching beyond the scope of a unit test.  Shouldn't it just expect that the `IndexPresenter#field_value` is called with the correct values?